src/data/store/fetch-all.js

```diff
@@ -0,0 +1,33 @@
+import QueryBuilder from '../query-builder'
+
+const MAX_PAGE_SIZE = 100 // max allowed value
```

Comment:
this parameter might be changed for PRO and Manage installations, I believe we could have some kind of configurations storage and give an ability to reconfigure it for the SDK

does it make sense?

Reply:
Done